conf/imperial.config

```diff
@@ -37,39 +37,39 @@ profiles {
             }
 
             withLabel:process_low {
-                queue  = 'v1_throughput72'
+                queue  = 'throughput72'
                 cpus   = { 2	 * task.attempt }
                 memory = { 48.GB * task.attempt }
                 time   = { 8 * task.attempt  }
             }
 
             withLabel:process_medium {
                 // TARGET QUEUE: throughput
-                queue = 'v1_throughput72'
+                queue = 'vthroughput72'
```

Comment:
Should this have a v at the start?

Reply:
Imperial HPC has recently moved away from specifying queue names. So, we need to remove all the queue names. 

In this following conf:

```
withLabel:process_low {
                queue  = 'throughput72'
                cpus   = { 2	 * task.attempt }
                memory = { 48.GB * task.attempt }
                time   = { 8 * task.attempt  }
            }
 ```
 
 The time should be  **8.h**